lib/node_modules/@stdlib/lapack/base/dlacpy/lib/base.js

```diff
@@ -0,0 +1,132 @@
+* @param {string} uplo - specifies whether the upper or lower triangular part of matrix `A` is supplied
+* @param {NonNegativeInteger} M - number of rows in matrix `A`
+* @param {NonNegativeInteger} N - number of columns in matrix `A`
+* @param {Float64Array} A - input matrix
+* @param {integer} strideA1 - stride of the first dimension of `A`
+* @param {integer} strideA2 - stride of the second dimension of `A`
+* @param {PositiveInteger} offsetA - starting index for `A`
+* @param {Float64Array} B - destination matrix
+* @param {integer} strideB1 - stride of the first dimension of `B`
+* @param {integer} strideB2 - stride of the second dimension of `B`
+* @param {PositiveInteger} offsetB - starting index for `B`
+* @returns {Float64Array} `B`
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* var A = new Float64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
+* var B = new Float64Array( 4 );
+*
+* dlacpy( 'row-major', 'all', 2, 2, A, 2, 1, 0, B, 2, 1, 0 );
+* // B => <Float64Array>[ 1.0, 2.0, 3.0, 4.0 ]
+*/
+function dlacpy( order, uplo, M, N, A, strideA1, strideA2, offsetA, B, strideB1, strideB2, offsetB ) { // eslint-disable-line max-len, max-params
+	var sa0;
+	var sa1;
+	var sb0;
+	var sb1;
+	var i0;
+	var i1;
+
+	if ( order === 'row-major' ) {
+		// For row-major matrices, the last dimension has the fastest changing index...
+		sa0 = strideA2; // stride for innermost loop
+		sa1 = strideA1; // stride for outermost loop
+		sb0 = strideB2; // stride for innermost loop
+		sb1 = strideB1; // stride for outermost loop
+	} else { // order === 'column-major'
+		// For column-major matrices, the first dimension has the fastest changing index...
+		sa0 = strideA1; // stride for innermost loop
+		sa1 = strideA2; // stride for outermost loop
+		sb0 = strideB1; // stride for innermost loop
+		sb1 = strideB2; // stride for outermost loop
+	}
+
+	if ( uplo === 'upper' ) {
+		if ( order === 'column-major' ) {
+			for ( i1 = 0; i1 < N; i1++ ) {
+				for ( i0 = 0; i0 < min( i1, M ); i0++ ) {
+					B[ offsetB + ( i0 * sb0 ) + ( sb1 * i1 ) ] = A[ offsetA + ( i0 * sa0 ) + ( sa1 * i1 ) ]; // eslint-disable-line max-len
+				}
+			}
+			return B;
+		}
+		// order === 'row-major'
+		for ( i1 = 0; i1 < N; i1++ ) {
+			for ( i0 = 0; i0 < min( i1, M ); i0++ ) {
+				B[ offsetB + ( i0 * sb1 ) + ( i1 * sb0 ) ] = A[ offsetA + ( i0 * sa1 ) + ( i1 * sa0 )]; // eslint-disable-line max-len
+			}
+		}
+		return B;
+	}
+	if ( uplo === 'lower' ) {
+		if ( order === 'column-major' ) {
+			for ( i1 = 0; i1 < N; i1++ ) {
+				for ( i0 = i1; i0 < M; i0++ ) {
+					B[ offsetB + ( i0 * sb0 ) + ( sb1 * i1 ) ] = A[ offsetA + ( i0 * sa0 ) + ( sa1 * i1 ) ]; // eslint-disable-line max-len
+				}
+			}
+			return B;
+		}
+		// order === 'row-major'
+		for ( i1 = 0; i1 < N; i1++ ) {
+			for ( i0 = i1; i0 < M; i0++ ) {
+				B[ offsetB + ( i0 * sb1 ) + ( i1 * sb0 ) ] = A[ offsetA + ( i0 * sa1 ) + ( i1 * sa0 )]; // eslint-disable-line max-len
+			}
+		}
```

Comment:
Same thing applies here. I am not sure we need to distinguish between row and column major provided loop interchange has been performed.